ssclj/jar/src/test/clojure/com/sixsq/slipstream/ssclj/resources/network_service_test.clj

```diff
@@ -0,0 +1,159 @@
+(ns com.sixsq.slipstream.ssclj.resources.network-service-test
+  (:require
+    [clojure.test :refer :all]
+    [peridot.core :refer :all]
+
+    [korma.core :as kc]
+
+    [com.sixsq.slipstream.ssclj.api.acl :as acl]
```

Comment:
@st [minor] In some files above `require`s are vertically aligned, and here they aren't. Personally, I like it vertically aligned, but anyway consistency should be present.